javatests/arcs/android/common/resurrection/ResurrectorServiceImpl.kt

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+package arcs.android.common.resurrection
+
+import android.content.Intent
+import android.os.IBinder
+
+class ResurrectorServiceImpl : ResurrectorService() {
```

Comment:
please, excuse my ignorance. i see this class only used in the Test code. Is there a follow up pr?

Reply:
This is just a test class. StorageService also implements ResurrectorService in this PR.

Reply:
(And tests are coming to verify that the StorageService performs resurrection, in the next PR)